client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/index.js

```diff
@@ -44,17 +45,16 @@ const renderSummary = ( props, showCountry ) => {
 	if ( ! isNormalized ) {
 		return translate( "You've edited the address, please revalidate it for accurate rates" );
 	}
-	const { countriesData } = storeOptions;
-	const { city, postcode, state, country } = getAddressValues( props );
+	const { city, postcode, state, country } = getAddressValues( addressData );
 	// Summary format: "city, state  postcode [, country]"
 	let str = city + ', ';
 	if ( state ) {
-		const statesMap = ( expandStateName && ( countriesData[ country ] || {} ).states ) || {};
-		str += ( statesMap[ state ] || state ) + '\xa0 '; // append two spaces: non-breaking and normal
+		const stateStr = expandStateName ? getStateName( reduxState, country, state, siteId ) : state;
```

Comment:
I wouldn't say this is really Redux state inherently, i.e. we could stop using Redux without modifying this code at all – what do you think of a more general name like `appState`?

Or maybe keeping it as `state` to have it consistent with the calling context, and qualifying the other "state" references instead?

Reply:
`appState` sounds better, thanks! Naming things is hard 😓